src/utils/setup/handle-create-page.js

```diff
@@ -12,23 +13,14 @@ let stitchClient;
 const MAX_LEARN_PAGE_FEATURED_ARTICLES = 3;
 const MAX_HOME_PAGE_FEATURED_ARTICLES = 4;
 
-const DEFAULT_FEATURED_HOME_SLUGS = [
-    'how-to/nextjs-building-modern-applications',
-    'how-to/python-starlette-stitch',
-    'how-to/graphql-support-atlas-stitch',
-    'quickstart/free-atlas-cluster',
-];
-
-const DEFAULT_FEATURED_LEARN_SLUGS = [
-    'quickstart/java-setup-crud-operations',
-    'how-to/golang-alexa-skills',
-    'how-to/polymorphic-pattern',
-];
-
 const memoizedStagingPages = memoizerific(1)(
     async () => await getStagingPages()
 );
 
+const memoizedBuildTimeMedia = memoizerific(1)(
```

Comment:
We are also going to be adding podcasts and videos in the near future so I pre-emptively added this memoization. It isn't used right now so I'm also happy to do this at a later time instead